cypress/integration/create_room_spec.js

```diff
@@ -0,0 +1,10 @@
+describe('Create room', () => {
+    beforeEach(() => cy.login('unicorn'));
+
+    describe('when a player has successfully logged in', () => {
+        it('is able to create a new room and is then added to that room', () => {
+            cy.get('[data-cy="create-room"]').click();
```

Comment:
Could you make a `cy.createRoom` command and have it wait for the request to respond like the others to prevent race conditions?

Reply:
sure